vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/IElementFilterAction.java

```diff
@@ -17,10 +17,28 @@
 package org.vividus.ui.action.search;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
+import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.WebElement;
 
 public interface IElementFilterAction extends IElementAction
 {
-    List<WebElement> filter(List<WebElement> elements, String value);
+    default UnsupportedOperationException unsupportedMatchesOperationException()
+    {
+        return new UnsupportedOperationException(
+            "The matches(WebElement, String) operation is not supported for the " + getType().getAttributeName()
+                + " filter");
```

Comment:
I don't think it should be in the interface, since it's the actual implementation
I propose to move it to the class level and even exception message can be skipped